examples/todo.html

```diff
@@ -29,7 +29,8 @@
             <py-tutor modules="./utils.py;./todo.py">
                 <py-config>
                     plugins = [
-                      "https://pyscript.net/latest/plugins/python/py_tutor.py"
+                      "https://pyscript.net/latest/plugins/python/py_tutor.py",
```

Comment:
This also should be fixed. We should not use latest.

Up to you whether to fix in this PR or in a follow up PR